tools/pom.xml

```diff
@@ -25,7 +23,7 @@
   <name>JBoss Fuse Tooling :: Tools</name>
 
   <modules>
-    <module>ide-catalog-generator</module>
+    <module>m2-repo-cleaner</module>
```

Comment:
on this branch the catalog-generator is still valid (until the "decouple branch" will be merged"